app/views/content/funding-and-support/if-youre-a-parent-or-carer.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Support training to teach if you're a parent or carer"
+heading: "Support training to teach if you're a parent or carer"
+description: |-
+  Find out about the extra grants and schemes you could be eligible for if you have children or other caring responsibilities.
+related_content:
+    Choosing the right teacher training course provider: "/blog/choosing-the-right-teacher-training-course-provider"
+    Get school experience: "/train-to-be-a-teacher/get-school-experience"
+    Diary of a trainee teacher: "/blog/a-diary-of-a-trainee-teacher"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-support-promo
+navigation: 20.25
+navigation_title: If you're a parent or carer
+navigation_description: Find out about the extra grants and schemes you could be eligible for if you have children or other caring responsibilities.
```

Comment:
@EllieNodder can we simplify/shorten slightly and change to something like: 'Find out what extra grants and schemes are available if you have children or other caring responsibilities.'